ci/build-unix.yml

```diff
@@ -92,6 +92,16 @@ steps:
       bazel build //ledger-service/http-json:http-json-binary_deploy.jar
       cp bazel-bin/ledger-service/http-json/http-json-binary_deploy.jar $(Build.StagingDirectory)/$JSON_API
       echo "##vso[task.setvariable variable=json-api;isOutput=true]$JSON_API"
+
+      TRIGGER=trigger-runner-${{parameters.release_tag}}.jar
+      bazel build //triggers/runner:trigger-runner_deploy.jar
+      cp bazel-bin/triggers/runner/trigger-runner_deploy.jar $(Build.StagingDirectory)/$TRIGGER
+      echo "##vso[task.setvariable variable=trigger-runner;isOutput=true]$TRIGGER"
+
+      SCRIPT=script-runner-${{parameters.release_tag}}.jar
```

Comment:
How do you feel about just calling this `daml-script-*.jar`? Matches the CLI command and the name of the feature. (The former also applies to the JSON API but the latter doesn’t so I think this doesn’t invalidate the point)

Reply:
No feeling on that, I'll make the change.